mihaela-etropolska/Homework 1/CV Mihaela Etropolska/mihaela-cv-style.css

```diff
@@ -0,0 +1,137 @@
+	background-size: 100%;	
+	background-repeat: no-repeat;
+}
+
+.info {
+	font-style: italic;
+	font-size: 1.2em;
+}
+
+.year {
+	margin-left: 200px;
+	background-color: black;
+	color: white;
+}
+
+.main-div {
+	margin: 50px auto;
+	width: 1300px;
```

Comment:
@mihaela-etropolska 
If you remove the width and height and add "overflow:auto" it will look much better : )